platform.json

```diff
@@ -37,6 +37,11 @@
       "optional": true,
       "version": "~5.51001.1"
     },
+    "framework-arduinoadafruitnordicnrf5": {
+      "type": "framework",
+      "optional": true,
+      "version": "https://github.com/adafruit/Adafruit_nRF52_Arduino.git"
```

Comment:
Need to make this an official Platform IO Package

Reply:
A package is deployed to our registry. Please use `1.000903.0` version.